noxfile.py

```diff
@@ -253,16 +237,30 @@ def make(session: nox.Session):
     """
     Generate HTML from documentation source using the Makefile
     """
-    parser = _relaxed_parser(session)
-    parser.add_argument(
-        "make_args", nargs="*", help="Specify make targets as arguments"
-    )
-    args = parser.parse_args(session.posargs)
+    make_args = session.posargs or ["clean", "coredocs"]
 
-    install(session, req="requirements-relaxed" if args.relaxed else "requirements")
+    install(session, req="requirements")
     _clone_core_check(session)
-    make_args: list[str] = [
+    session.run(
+        "make",
+        "-C",
+        "docs/docsite",
         f"PYTHON={_env_python(session)}",
-        *(args.make_args or ("clean", "coredocs")),
-    ]
-    session.run("make", "-C", "docs/docsite", *make_args, external=True)
+        *make_args,
+        external=True,
+    )
+
+
+@nox.session
+def tag(session: nox.Session):
```

Comment:
I think this was intentionally not there. (And maybe should also be removed in stable-2.18? Or should we just keep it to make branching simpler?)

Reply:
good catch. the tagger shouldn't be on this branch. it looks like it was already on stable-2.18 but I'll send another PR to remove it. I'll also check the docs in the readme so it's clear the tag stuff should be on devel only. thanks again for spotting this.